problem-16.c

```diff
@@ -0,0 +1,82 @@
+#include <stdio.h>
+#include <stdbool.h>
+// Basic calculator program that performs addition, subtraction, multiplication and division
+
+int getSum(int firstNum, int secondNum);
+int getDifference(int firstNum, int secondNum);
+int getProduct(int firstNum, int secondNum);
+int getQuotient(int firstNum, int secondNum);
+
+int main()
+{
+    int firstNum, secondNum, user_choice;
+    printf("---------CALCULATOR---------\n");
+
+    printf("First number = ");
+    scanf("%d", &firstNum);
+
+    printf("Second Number = ");
+    scanf("%d", &secondNum);
+
+    printf("\n1. Addition");
+    printf("\n2. Subtraction");
+    printf("\n3. Multiplication");
+    printf("\n4. Division");
+
+    while (true)
+    {
+        printf("\n\nEnter your choice: ");
+        scanf("%d", &user_choice);
+
+        if (user_choice == 1)
+        {
+            printf("\nThe sum of %d and %d is %d", firstNum, secondNum, getSum(firstNum, secondNum));

```

Comment:
Better to match the output message with the menu that we are showing to the users.